tinker/main.py

```diff
@@ -12,9 +12,9 @@
 import sys
 from typing import List, Set, Union, Type
 
-from . import algorithm
-from . import protocol
-from .configuration import parse_configuration
+from tinker import algorithm
+from tinker import protocol
+from tinker.configuration import parse_configuration
```

Comment:
The correct thing to do is actually to use the relative imports, unless you have some reason to need to change it.

Reply:
Fixed in https://github.com/tinker-engine/tinker-engine/commit/fc7f1a0195ce3ad089f6566d23227a43443218cb